api/src/main/java/org/pmiops/workbench/api/DataSetController.java

```diff
@@ -352,34 +345,76 @@ public DomainValuePair apply(DataSetValues dataSetValue) {
       String workspaceNamespace, String workspaceId, DataSetExportRequest dataSetExportRequest) {
     workspaceService.getWorkspaceEnforceAccessLevelAndSetCdrVersion(
         workspaceNamespace, workspaceId, WorkspaceAccessLevel.WRITER);
-    DataSetQueryList queryList =
-        generateQuery(workspaceNamespace, workspaceId, dataSetExportRequest.getDataSetRequest())
-            .getBody();
-    String queriesAsStrings =
-        "import pandas\n\n"
-            + queryList.getQueryList().stream()
-                .map(
-                    query ->
-                        convertQueryToString(
-                            query, dataSetExportRequest.getDataSetRequest().getName()))
-                .collect(Collectors.joining("\n\n"));
-    JSONObject notebookFile;
+    // This suppresses 'may not be initialized errors. We will always init to something else before
+    // used.
+    JSONObject notebookFile = new JSONObject();
```

Comment:
Not blocking this PR, but a philosophical question: is it a common AOU pattern to use generic JSON objects over more explicit types?

Reply:
Talked offline. Not explicitly decided as such, but that pattern does sort of exist. It may make sense to revisit that pattern, but outside the scope of this PR.